Polyhedron/demo/Polyhedron/Plugins/Point_set/Features_detection_plugin.cpp

```diff
@@ -1,3 +1,4 @@
+#include <QAction>
```

Comment:
Actually, that new error in ArchLinux is not related to this PR. ArchLinux has upgraded Qt from 5.10 to 5.11.

There are other compilation errors, in the demos of arrangements and linear
cell complex:
```
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:28:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:32:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:34:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:36:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:38:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:40:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:48:27: error: invalid use of incomplete type 'class QButtonGroup'
Linear_cell_complex_Demo/MainWindow.cpp:55:31: error: invalid use of incomplete type 'class QHeaderView'
Linear_cell_complex_Demo/MainWindow.cpp:70:33: error: invalid use of incomplete type 'class QHeaderView'
Linear_cell_complex_Demo/MainWindow.cpp:70:69: error: incomplete type 'QHeaderView' used in nested name specifier
```